### PR TITLE
Cha221/play pause

### PR DESCRIPTION
**Pull request by cha2251**
* Added methods to update Mixing system's state when buttons are pushed
* Added play, stop, pause, and unpause methods
* Hooked methods into UI buttons

### CODE REVIEW

File: src/button_input/button_input.py [hunk 1]
```diff
@@ -2,8 +2,8 @@
 import time
 from threading import Thread
 from pynput import keyboard
-
 import mido
+import time
```
Comment (DavidTheFighter) [inline src/button_input/button_input.py:6]: Duplicate `import time`

File: src/common/main.py [hunk 2]
```diff
@@ -12,6 +12,7 @@
 import mido
 import mido.backends.rtmidi  # Needed for windows builds w/ pyinstaller
 
+CONSOLE_MODE = True # Flip to enable devleoping in console mode
```
Comment (DavidTheFighter) [inline src/common/main.py:15]: Maybe change the comment on this a bit so people realize this doesn't disable the UI, this took me a sec to process.

File: src/common/main.py [hunk 3]
```diff
@@ -40,29 +41,38 @@ def main(self):
...
-        x = Thread(target=self.init_UI, args=(self.shutdown,))
+        x = Thread(target=self.init_UI, args=(self.shutdown,self.mixing))
         x.start()
 
-
         print("Type `quit` to quit")
 
-        while(True):
-            if (False and input() == 'quit') or not x.is_alive():  # Change false when developing in console mode
+        while(CONSOLE_MODE and x.is_alive()):
```
Comment (DavidTheFighter) [inline src/common/main.py:53]: What happens when CONSOLE_MODE is false? Wouldn't the system shut down prematurely?

**Review by DavidTheFighter — CHANGES_REQUESTED** [review]
Just a few small things that I can't help but nit pick :D

File: src/common/main.py (hunk 3 above)
Comment (cha2251) [reply src/common/main.py:53]: tiny insignificant detail

**Review by DavidTheFighter — APPROVED** [review]
LGTM!

**Pull request merged**